### DISCUSSION TITLE
What happened to the ViewBox control in Avalonia 11?

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#12717] [Q&A]
I'm trying to use a ViewBox control in Avalonia 11, and I get the following error:

AVLN:0004	Unable to resolve type ViewBox from namespace https://github.com/avaloniaui 

The ViewBox still works in version 10.

Do I have to add reference somewhere?

**Comment by stevemonaco** [comment]
Should be available out-of-the-box. I just created a project from the Avalonia MVVM App template and it works fine:

```xml
<Viewbox>
  <TextBlock Text="{Binding Greeting}" HorizontalAlignment="Center" VerticalAlignment="Center"/>
</Viewbox>
```

**Comment by maxkatz6** [comment]
> ViewBox

It's `Viewbox` and always was.

**Comment by wgraaf** [comment]
Oops. I guess the docmentation was off (https://docs.avaloniaui.net/docs/controls/viewbox):

![image](https://github.com/AvaloniaUI/Avalonia/assets/123667061/dd3dbfbf-d843-447a-ba8e-baaa17aee66a)

**Reply by maxkatz6** [reply]
Thanks. Fixed casing on this page.
